### ISSUE TITLE
create cli flags for install_gems

### ISSUE THREAD

**Issue by OpenNebulaProject** [OpenNebula/one#1162]
---

Author Name: **Rogier Mars** (Rogier Mars)
Original Redmine Issue: 4980, https://dev.opennebula.org/issues/4980
Original Date: 2017-01-12

---

Hi, 

Would it be possible to create flags to set the OS and make the script non-interactive? This would make it easier to run the script from configmanagement like ansible. Now we have to wrap it in expect, and that causes other issues. 

For example:
```
install_gems --os=centos --silent --force
install_gems --os=ubuntu --silent --force

```

**Comment by kvaps** [comment]
Agree, for now I use this workaround:

```
sed -i 's/apt-get install/apt-get -y install/g' /usr/share/one/install_gems
echo '0\n' | /usr/share/one/install_gems
```

**Comment by vholer** [comment]
Fixed by #4233, `install_gems` is not interactive anymore.
